packages/reference/src/__fixtures__/FakeSdk.ts

```diff
@@ -25,14 +25,16 @@ const newLink = (linkType: string, id: string): Link => ({
 
 // used for component testing
 export type ReferenceEditorSdkProps = {
+  initialValue?: any;
   validations?: any;
+  shouldDelay?: boolean;
```

Comment:
`shouldDelay` isn't descriptive enough of precisely what will be delayed. e.g. why only delay asset & entry calls while we also have space, dialogs ..etc. It also doesn't allow modifying the delay value.